### PR TITLE
Snowflake/z otp

### PR DESCRIPTION
**Pull request by saldomik**
Snowflake z-otp component, necessary for new signup flow.

### CODE REVIEW

File: src/components/inputs/z-otp/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <div class="otp-container">
+        {this.otp.map((_elem, i) => {
+          return (
+            <input
+              onInput={(e: any) => {
+                this.otp[i] = e.target.value;
+                this.emitInputChange(this.otp.join(""));
+              }}
+              onKeyUp={(e: any) => {
+                if (e.keyCode !== 8 && e.keyCode !== 46)
```
Comment (vpasquino) [inline src/components/inputs/z-otp/index.tsx:39]: You should update `keybordKeyCodeEnum` in `src/beans/index.tsx` and use it here

File: src/components/inputs/z-otp/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                  i < this.inputNum && this.otpRef[i + 1].focus();
+              }}
+              type="tel"
```
Comment (vpasquino) [inline src/components/inputs/z-otp/index.tsx:42]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel
`tel` should be a phone number, maybe `number` would be better?

File: src/components/inputs/z-otp/styles.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  width: 40px;
+  font-size: 40px;
+  text-align: center;
+  caret-color: transparent;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+input:not(:last-of-type) {
+  margin-right: 16px;
```
Comment (vpasquino) [inline src/components/inputs/z-otp/styles.css:17]: Use css variable

File: src/components/inputs/z-otp/index.tsx (hunk 2 above)
Comment (saldomik) [reply src/components/inputs/z-otp/index.tsx:42]: input number has a really ugly standard layout :sweat_smile: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

Comment (vpasquino) [reply src/components/inputs/z-otp/index.tsx:42]: I know, but you can hide the arrows :sweat_smile:

**Pull request merged**